modules/odc-grouping-multiple-components-within-an-application.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// applications/application-life-cycle-management/odc-viewing-application-composition-using-the-topology-view.adoc
+
+[id="odc-grouping-multiple-components-within-an-application_{context}"]
+= Grouping multiple components within an application
+
+You can use the *Add* page to add multiple components or services to your project and use the *Topology* page to group applications and resources within an application group.
+For this tutorial, add a MongoDB database service to an existing application with a Node.js component as follows:
```

Comment:
Consider rewriting as:
The following procedure adds a MongoDB database service to an existing application with a Node.js component.